server/visibility/queries.ts

```diff
@@ -23,9 +18,36 @@ export const getParentModelForVisibility = async (
 	return null;
 };
 
+type UpdateOptions = {
+	visibilityId: string;
+	access: types.Visibility['access'];
```

Comment:
hadn't realized (remembered?) you could access type properties (that is, sub-types?) this way. Oh, Typescript, you never cease to humble.